stories/ImagesList/ImagesList.stories.tsx

```diff
@@ -0,0 +1,666 @@
+import React from 'react';
+import { Icon } from '../../src';
+
+export const ElementoSingolo = () => (
+  <div className='row'>
+    <div className='col-12 col-lg-4'>
+      <div className='it-grid-item-wrapper'>
+        <a href='#'>
+          <div className='img-responsive-wrapper'>
+            <div className='img-responsive'>
+              <div className='img-wrapper'>
+                <img
+                  src='https://via.placeholder.com/660x300/ebebeb/808080/?text=Immagine'
+                  alt='Alternative Text'
+                  title='Image Title'
+                />
+              </div>
+            </div>
+          </div>
```

Comment:
The `HeroBackground` component has a very similar structure: perhaps it can be refactored/renamed into a `ResponsiveImage` component? (Thus update the `Hero` examples)

Reply:
Created. Do you think it is necessary to create the relevant documentation?

Reply:
Maybe just update the Hero one.
Do you mind open an issue about documenting this new component?
We can save it for later